colaraz/lms/templates/header/navbar-authenticated.html

```diff
@@ -13,6 +13,14 @@
   show_explore_courses = settings.FEATURES.get('COURSES_ARE_BROWSABLE') and not show_program_listing
   show_sysadmin_dashboard = settings.FEATURES.get('ENABLE_SYSADMIN_DASHBOARD','') and user.is_staff
   self.real_user = getattr(user, 'real_user', user)
+
+  if user.is_authenticated:
+    invite_friend_url = '{}://{}.{}/Home/InviteFriends.aspx'.format(
```

Comment:
nit: What if we make a configurable constant for the URL ?

Reply:
nit